docs/source/sysadmin_guide/configure_sgx.rst

```diff
@@ -417,13 +427,102 @@ specify the correct bind interface.
         Jun 02 14:50:37 ubuntu systemd[1]: sawtooth-validator.service: Unit entered failed state.
         Jun 02 14:50:37 ubuntu systemd[1]: sawtooth-validator.service: Failed with result 'exit-code'.
 
-Restrict permssions on ``validator.toml`` to protect the network private key.
+(Optional) Change the network keys to specify secured network communication
+between nodes in the network. By default, the network is unsecured.
+
+Locate the ``network_public_key`` and ``network_private_key`` settings.
+These items specify the curve ZMQ key pair used to create a secured
+network based on side-band sharing of a single network key pair to all
+participating nodes.
+
+Next, generate your network keys.
+
+ * This example shows how to use Python to generate these keys:
+
+   .. code-block:: python
+
+       python
+        ...
+       >>> import zmq
+       >>> (public, secret) = zmq.curve_keypair()
+       >>> print public
+       wFMwoOt>yFqI/ek.G[tfMMILHWw#vXB[Sv}>l>i)
+       >>> print secret
+       r&oJ5aQDj4+V]p2:Lz70Eu0x#m%IwzBdP(}&hWM*
+
+ * Or you could use the following steps to compile and run ``curve_keygen``
+   to generate the keys:
+
+   .. code-block:: console
+
+       $ sudo apt-get install g++ libzmq3-dev
```

Comment:
it looks like there is an extra space on this line compared to the lines below.

Reply:
Fixed.  Thanks!